doc/lightning_gpu/device.rst

```diff
@@ -165,9 +162,12 @@ The workflow for the multi-node/GPUs feature is as follows:
         return qml.state()
     local_state_vector = circuit_mpi()
 
+.. note::
+    The total number of MPI processes and MPI processes per node must be powers of 2. For example, 2, 4, 8, 16, etc. Each MPI process is responsible for managing one GPU. 
+
 Currently, a ``lightning.gpu`` device with the MPI multi-GPU backend supports all the ``gate operations`` and ``observables`` that a ``lightning.gpu`` device with a single GPU/node backend supports.
 
-By default, each MPI process will return the overall simulation results, except for the ``qml.state()`` and ``qml.prob()`` methods for which each MPI process only returns the local simulation
+By default, each MPI process will return the overall simulation results, except for the ``qml.state()`` and ``qml.probs()`` methods for which each MPI process only returns the local simulation
 results for the ``qml.state()`` and ``qml.prob()`` methods to avoid buffer overflow. It is the user's responsibility to ensure correct data collection for those two methods. Here are examples of collecting
```

Comment:
🫠 
```suggestion
results for the ``qml.state()`` and ``qml.probs()`` methods to avoid buffer overflow. It is the user's responsibility to ensure correct data collection for those two methods. Here are examples of collecting
```